a9_fake_news_simulator/environment.py

```diff
@@ -96,8 +96,10 @@ def agent_conversation(self, agent_a, agent_b):
                 # resolve agent acceptance
                 if winner == 1:
                     agent_b.evaluate_opinion(agent_a.opinion)
+                    agent_b.opinion_base[self.agent_list.index(agent_a)] = agent_a.opinion
```

Comment:
The question whether this line should be in the if block or outside of it makes a difference between agent a calling agent b until they were finally convinced in a discussion, or agent a calling agent b only when they never head a discussion.